src/main/kotlin/no/nav/familie/ks/sak/integrasjon/journalføring/InnkommendeJournalføringService.kt

```diff
@@ -46,18 +47,19 @@ class InnkommendeJournalføringService(
     private val loggService: LoggService,
     private val behandlingSøknadsinfoService: BehandlingSøknadsinfoService,
 ) {
-    fun hentJournalposterForBruker(brukerId: String): List<Journalpost> =
-        integrasjonClient.hentJournalposterForBruker(
-            JournalposterForBrukerRequest(
-                antall = 1000,
-                brukerId =
-                    Bruker(
-                        id = brukerId,
-                        type = BrukerIdType.FNR,
-                    ),
-                tema = listOf(Tema.KON),
-            ),
-        )
+    fun hentJournalposterForBruker(brukerId: String): List<TilgangsstyrtJournalpost> =
+        integrasjonClient
+            .hentTilgangsstyrteJournalposterForBruker(
+                JournalposterForBrukerRequest(
+                    antall = 1000,
+                    brukerId =
```

Comment:
Kanskje ikke relatert til det du jobber med, men er det rart at dette parameteret heter `brukerId` og tar i mot et `Bruker` objekt`? Hadde egentlig forventet et `String` eller noe for `brukerId`